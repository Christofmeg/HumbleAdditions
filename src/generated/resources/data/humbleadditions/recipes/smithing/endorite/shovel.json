{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorite_block"
  },
  "base": {
    "item": "humbleadditions:endorium_shovel"
  },
  "result": {
    "item": "humbleadditions:endorite_shovel"
  }
}