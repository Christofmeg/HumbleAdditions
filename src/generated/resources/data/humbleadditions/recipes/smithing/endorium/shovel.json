{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "minecraft:diamond_shovel"
  },
  "result": {
    "item": "humbleadditions:endorium_shovel"
  }
}