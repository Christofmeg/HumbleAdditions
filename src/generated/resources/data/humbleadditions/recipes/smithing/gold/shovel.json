{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_block"
  },
  "base": {
    "item": "minecraft:stone_shovel"
  },
  "result": {
    "item": "minecraft:golden_shovel"
  }
}