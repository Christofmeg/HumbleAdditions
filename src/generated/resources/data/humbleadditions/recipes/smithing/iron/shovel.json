{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_block"
  },
  "base": {
    "item": "minecraft:golden_shovel"
  },
  "result": {
    "item": "minecraft:iron_shovel"
  }
}