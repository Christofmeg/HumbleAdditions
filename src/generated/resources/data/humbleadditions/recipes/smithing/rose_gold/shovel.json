{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_ingot"
  },
  "base": {
    "item": "minecraft:golden_shovel"
  },
  "result": {
    "item": "humbleadditions:rose_gold_shovel"
  }
}