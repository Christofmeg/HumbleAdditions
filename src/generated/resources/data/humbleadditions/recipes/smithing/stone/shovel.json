{
  "type": "minecraft:smithing",
  "addition": {
    "tag": "minecraft:stone_tool_materials"
  },
  "base": {
    "item": "minecraft:wooden_shovel"
  },
  "result": {
    "item": "minecraft:stone_shovel"
  }
}