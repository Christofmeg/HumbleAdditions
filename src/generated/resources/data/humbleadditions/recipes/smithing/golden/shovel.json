{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "minecraft:stone_shovel"
  },
  "result": {
    "item": "minecraft:golden_shovel"
  }
}